src/sentry/grouping/ingest/config.py

```diff
@@ -99,5 +99,5 @@ def project_uses_optimized_grouping(project: Project) -> bool:
             project.organization,
         )
         or (is_in_transition(project))
-        or project.id % 5 < 3  # 60% of all non-transition projects
+        or project.id % 5 < 4  # 60% of all non-transition projects
```

Comment:
small nit: change the comment to 80%

Reply:
Yup, I realized that as soon as I came back to look at it after pasting the link in slack. Fixed.